dual_manipulator:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 125

  arm_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - joint_1_1
      - joint_2_1
      - joint_3_1
      - joint_4_1
      - joint_5_1
      - joint_6_1
      - joint_1_2
      - joint_2_2
      - joint_3_2
      - joint_4_2
      - joint_5_2
      - joint_6_2